flake.nix

```diff
@@ -181,7 +181,14 @@
           mkdir -p $out
         '';
 
-        installCheckPhase = "make installcheck -j$NIX_BUILD_CORES -l$NIX_BUILD_CORES";
+        installCheckPhase = ''
+          export PATH=${runCommand "nix-2.3-prefetch" { } ''
+            mkdir -p $out/bin
+            cp ${nixStable}/bin/nix-prefetch-url $out/bin/nix-prefetch-url-2.3
+            $out/bin/nix-prefetch-url-2.3 --version | grep -q 2.3
+          ''}/bin:$PATH
+          make installcheck -j$NIX_BUILD_CORES -l$NIX_BUILD_CORES
```

Comment:
This seems very complicated. `testNixVersions` already takes a `client` and `daemon` version, and now we're adding *another* version into the mix (`nixStable`), passed in a very hacky way (by adding a `nix-2.3-prefetch` derivation to `PATH`).

Shouldn't the client/daemon matrix be able to test this? (E.g. by enabling `againstLatestStable`)

Reply:
Yeah, that should certainly be possible and as mentioned above, I mostly hacked this together to get a working regression test for others :) 

Is the `testNixVersions` thing somehow easily usable for dev purposes btw? :) I'd still like to be able to test something like this `make tests/foobar.sh.test`.

Reply:
@Ma27 yes it is. You can define `NIX_{DAEMON,CLIENT}_PACKAGE` yourself.